index.html

```diff
@@ -14,6 +14,29 @@
 </head>
 
 <body>
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
```

Comment:
- [ ] **Please make sure that the current date is displayed below the navigation bar.**

This is a project requirement and is also included in the wireframe. Please make sure to use the Luxon library.

![image](https://user-images.githubusercontent.com/59168430/124807466-6a296000-df7b-11eb-92e6-678aad83d4d6.png)

Reply:
Hi @smcommits, how are you? 
I'm using this luxon library but it fails ESLinters, it more than 1000 errors, how can I sort this out?

Thank you.

Reply:
I added luxon.js to .eslintignore to avoid those errors.